mihail.rychkov/task-2-2/pkg/primheap/external.go

```diff
@@ -0,0 +1,60 @@
+package primheap
+
+import (
+	"cmp"
+	"container/heap"
+	"errors"
+)
+
+type PrimHeap[T cmp.Ordered] struct {
+	heap unwrappedPrimHeap[T]
```

Comment:
```suggestion
	unwrappedPrimHeap[T]
```

тогда, например `Len`, `Push` и так далее можно будет просто унаследовать...